src/main/java/com/minecolonies/coremod/blocks/BlockTimberFrame.java

```diff
@@ -87,6 +88,7 @@ public MapColor getMapColor(@NotNull final IBlockState state)
      */
     @NotNull
     @Override
+    @Deprecated
     public IBlockState getStateFromMeta(final int meta)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)